test/serializer/additional-functions/noconflict-existantobject.js

```diff
@@ -0,0 +1,21 @@
+// additional functions
+// does not contain:x = 5;
+// does not contain:y = 10;
+
+global.c = {};
+
+function additional1() {
+  global.c.foo = 5;
+  var x = 5;
+}
+
+function additional2() {
+  global.c.bar = 2;
+  var y = 10;
+}
+
+inspect = function() {
+  additional2();
+  additional1();
+  return global.b + JSON.stringify(global.c);
```

Comment:
This test case currently fails because we don't serialize effects pertaining to existent objects aside from the ones that are needed by the generator from Effects.